### ISSUE TITLE
Wrong date formatting in agendaweek with German locale

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#2504]
Originally reported on Google Code with ID 2239

```
If the language is set to German, then the heading of agendaweek is missing a dot after
the month number. 

See here http://jsfiddle.net/4opj6djj/4/ 

It should be "Di. 12.08." (which is with trailing dot)


Thank you in advance for fixing this! :-)
```

Reported by `mikejones@xmail.net` on 2014-08-12 12:52:39

Imported with **4** stars.

**Comment by mlandes** [comment]
This is because of the "longDateFormat" definition in the moment locale files...
=> longDateFormat: LL, LLL, LLLL, ll, lll, llll 
The moment day format "Do" should be used instead of "D."
See: https://github.com/moment/moment/blob/develop/locale/de.js
and here a correct definition: https://github.com/moment/moment/blob/develop/locale/fi.js

**Comment by arshaw** [comment]
Fixed in [v3.2.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v3.2.0)
